shared/config/dependencies.sh

```diff
@@ -11,3 +11,4 @@ export SHELLCHECK_VERSION="v0.8.0"
 export TEKTONCD_CLI_VERSION="v0.27.0"
 export YAMLLINT_VERSION="1.28.0"
 export YQ_VERSION="v4.28.1"
+export BITWARDEN_VERSION="cli-v2022.10.0/bw-linux-2022.10.0"
```

Comment:
That should probably be `v2022.10.0`, and the rest should be part of the curl url.

Reply:
Done.